src/app/types/gameobject-template.type.ts

```diff
@@ -0,0 +1,50 @@
+import { TableRow } from './general';
+
+export const GAMEOBJECT_TEMPLATE_TABLE = 'gameobject_template';
+export const GAMEOBJECT_TEMPLATE_ID = 'entry';
+export const GAMEOBJECT_TEMPLATE_NAME = 'name';
+export const GAMEOBJECT_TEMPLATE_CUSTOM_STARTING_ID = 9_000_000;
```

Comment:
the current highest **entry** value in `gameobject_template` is `400_000` so we could make the custom start from `900_000` (one less zero)